diff -ur festival.org/src/modules/clunits/clunits.h festival/src/modules/clunits/clunits.h
--- festival.org/src/modules/clunits/clunits.h	2001-07-05 12:19:09.000000000 +0000
+++ festival/src/modules/clunits/clunits.h	2006-04-01 20:36:30.552506750 +0000
@@ -83,9 +83,9 @@
     CLfile *get_fileitem(const EST_String &name)
 	{ return (CLfile *)fileindex.lookup(name); }
     void load_coefs_sig(EST_Item *unit);
-    CLfile *CLDB::get_file_coefs_sig(const EST_String &fileid);
+    CLfile *get_file_coefs_sig(const EST_String &fileid);
     void load_join_coefs(CLunit *unit);
-    CLfile *CLDB::get_file_join_coefs(const EST_String &fileid);
+    CLfile *get_file_join_coefs(const EST_String &fileid);
 };
 
 
