--- festival/lib/init.scm~	2004-07-16 01:03:04.000000000 +0200
+++ festival/lib/init.scm	2012-07-17 22:18:30.268474432 +0200
@@ -77,6 +77,8 @@
 (cond
  ((member 'nas *modules*)
   (Parameter.def 'Audio_Method 'netaudio))
+ ((member 'pulseaudio *modules*)
+  (Parameter.def 'Audio_Method 'pulseaudio))
  ((member 'esd *modules*)
   (Parameter.def 'Audio_Method 'esdaudio))
  ((member 'sun16audio *modules*)
